pydov/types/abstract.py

```diff
@@ -53,7 +53,7 @@ def _parse(cls, func, xpath, namespace, returntype):
 
         """
         if namespace is not None:
-            ns = '{%s}' % namespace
+            ns = '{{{}}}'.format(namespace)
```

Comment:
New to me, why the triple accolades?

Reply:
To get a literal `{` you have to escape it by doubling. The middle pair is substituted by `namespace`. Looks a bit odd, but this sequence of `{{{}}}` is to get `{namespace}`. See also https://docs.python.org/3.7/library/string.html#format-string-syntax

Reply:
so, if updating to new string formatting, why not 

```
ns = f'{{namespace}}'
```

Reply:
Because that syntax is only supported as of Python 3.6.

Reply:
Ok, so I guess it's about time that I drop my request for Python 2.7 support :-)